core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -186,6 +186,9 @@ public int getPosition() {
 
     public void setPosition(int position) {
         this.position = position;
+        if(position > 0 && item.isNew()) {
+            this.item.setPlayed(false);
```

Comment:
If the media has been partially played (position > 0) and it has formerly been flagged as new, we mark is as unplayed (=``setPlayed(false)``)
I find this the best place to change from ``new`` to ``unplayed``, while smart mark as played is currently done on different events like starting to play another episode